ironfish/src/wallet/account.ts

```diff
@@ -505,10 +508,21 @@ export class Account {
     blockHash: Buffer | null
     sequence: number | null
   }> {
+    const head = await this.getHead()
+    if (!head) {
+      return {
+        unconfirmed: BigInt(0),
+        confirmed: BigInt(0),
```

Comment:
You can use literal syntax too if you want

```suggestion
        unconfirmed: 0n,
        confirmed: 0n,
```